src/OrleansAzureUtils/Storage/AzureStorageUtils.cs

```diff
@@ -386,5 +389,38 @@ internal static string PrintStorageException(Exception exception)
                         (extendedError.AdditionalDetails != null && extendedError.AdditionalDetails.Count > 0) ?
                             String.Format(", ExtendedErrorInformation.AdditionalDetails = {0}", Utils.DictionaryToString(extendedError.AdditionalDetails)) : String.Empty);
         }
+
+        internal static JsonSerializerSettings ConfigureJsonSerializerSettings(IProviderConfiguration config)
```

Comment:
Should we make this method public, so ANY provider can use it, and not the privileged ones in AzureStorageUtils?
I was suggesting that in the past, but got push back, also since I was not cloning.
Making it public will set on the path of fulfilling III in https://github.com/dotnet/orleans/issues/1495.